src/components/Partner/Partner.jsx

```diff
@@ -0,0 +1,15 @@
+export const Partner = ({ person }) => {
+  const { partnerName, sex, isMarried } = person;
+
+  return (
+    <>
+      {isMarried ? (
+        <p className="Person__partner">
```

Comment:
Also change class name in jsx to lowercase

Reply:
answer is here https://github.com/mate-academy/react_person/pull/2493#discussion_r1692218728